pkg/manifests/manifests.go

```diff
@@ -2054,3 +2092,40 @@ func NewSecurityContextConstraints(manifest io.Reader) (*securityv1.SecurityCont
 
 	return &s, nil
 }
+
+func (f *Factory) TelemeterConfigmapHash(caBundleCM *v1.ConfigMap) (*v1.ConfigMap, error) {
+	// Copy the CA Bundle ConfigMap to new CM but change name to append the hash.
+	// ca-bundle.crt is the key that.
+
+	data := make(map[string]string)
+	r := newErrMapReader(data)
```

Comment:
just a nit: as a) not having a key here is indeed expected and b) we're just reading out just one value, it is not really necessary to use the `errMapReader`. We can use the simple native Go map instead.

Reply:
Yep noticed as well, see comment below :D